### PR TITLE
Add Contact section & Footer section #4

### PR DESCRIPTION
**Pull request by MontaserSaleh91**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,11 +4,45 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="style.css" rel="stylesheet">
+    <script src="https://use.fontawesome.com/23bea62652.js"></script>
     <title></title>
 </head>
 <body>
     <div class="main-container" >
-         
-    </div>
+         <section class="contact-section">
```
Comment (YakoobHammouri) [inline index.html]: `div class="main-container"`  must be inside section

File: style.css [hunk 2]
```diff
@@ -39,15 +40,132 @@ h1{
...
+    flex-direction: column;
+    /* padding: 30px 0 50px 0; */
+    text-align: center;
+    font-family:'palatino linotype',serif;
+    color: #3D3D3D;
+    align-items: center;
+}
+
+.section-title{
+    width: 750px;
```
Comment (YakoobHammouri) [inline style.css]: do not use fix width

File: style.css [hunk 3]
```diff
@@ -39,15 +40,132 @@ h1{
...
+    margin-top: 67px;
+    margin-right: 0px;
+    margin-bottom: 41px;
+    padding: 50px;
+    border-bottom:1px solid rgba(83, 134, 228, 1);
+    font-family: 'spinnaker', sans-serif;
+    color: #3D3D3D;
+}
+
+.contact{
```
Comment (YakoobHammouri) [inline style.css:43]: this class name using by @anaal  and @sarah-alsharif0

**Review by YakoobHammouri — CHANGES_REQUESTED** [review]

**Review by YakoobHammouri — APPROVED** [review]

**Review by sarah-alsharif0 — APPROVED** [review]

**Comment by MontaserSaleh91** [comment]
- add contact title <h2>
- add paragraph
- create subscribe dev contains email input & button
- add urls as anchor links 
..1. horizontally for media query when screen bigger than 600px
.. 2. vertical for media query when screen less than 600px
- add font awesome icons for social media in follow me dev
- create footer that contains copyright 

image
image

File: index.html (hunk 1 above)
Comment (MontaserSaleh91) [reply index.html]: done

**Pull request merged**